scripts/config-excursion.json

```diff
@@ -23,6 +23,194 @@
           }
         }
       ]
+    },
+    "SCSSVariables": {
+      "transformGroup": "scss",
+      "prefix": "ds",
+      "comment": "File type: SCSS; variable type: Sass; filter: public",
+      "buildPath": "./dist/tokens/excursion/",
+      "files": [
+        {
+          "destination": "SCSSVariables.scss",
+          "format": "scss/variables",
+          "filter": {
+            "public": true
+          }
+        },
+        {
+          "destination": "SCSSVariablesMapFlat.scss",
+          "format": "custom/scss/map-flat",
+          "mapName": "auroTokens",
+          "filter": {
+            "public": true
+          }
+        }
+      ]
+    },
+    "SCSSVariableMap": {
+      "transformGroup": "scss",
+      "prefix": "ds",
+      "comment": "File type: SCSS; variable type: Sassmap; filter: size, public",
+      "dependency": "WCSS: ./src/utilityMixins/_spacingUtility.scss",
+      "buildPath": "./dist/tokens/excursion/",
+      "files": [
+        {
+          "destination": "SCSSVariableMap.scss",
+          "format": "scss/map-deep",
+          "filter": {
+            "attributes": {
+              "category": "size"
+            },
+            "public": true
+          }
+        }
+      ]
+    },
+
+    "CSSCustomProperties": {
```

Comment:
This is the only "new" platform that needs to be exported for Excursion. Remove all the others. That should leave the two pre-existing JSON platforms and the CSSCustomProperties platform.

Reply:
All right, only the json and css.